### PR TITLE
Add new link to user guide closes #547

### LINKED ISSUE #547: Add a new link to User Guide
I've moved the documentation for end users to this repo:

https://github.com/torrust/torrust-index-gui-user-guide

It's deployed to GitHub pages here:

https://torrust.github.io/torrust-index-gui-user-guide/

For the time being, it is elementary, but we are planning to add more sections.

It would be helpful to add a new "Help" link to this menu:

image

So users can easily find the documentation.

### PR DESCRIPTION
**Pull request by anujkumardarji**
The "Help" option has been added as a link above the logout option in the user options located at the top right corner.

### CODE REVIEW

**Comment by josecelano** [comment]
Thank you @anujkumardarji !!

image

File: cypress/e2e/contexts/torrent/specs/list/magnet_link.cy.ts
Comment (josecelano) [inline cypress/e2e/contexts/torrent/specs/list/magnet_link.cy.ts:1]: @anujkumardarji, there is a broken E2E test due to these changes.

https://github.com/torrust/torrust-index-gui/actions/runs/8893896657/job/24507852014?pr=548

**Review by josecelano — CHANGES_REQUESTED** [review]
Hi @anujkumardarji I do not know why you changed the Cypress test. Could you revert that change to merge this PR? Or explain why it's needed and fix the test.

File: cypress/e2e/contexts/torrent/specs/list/magnet_link.cy.ts
Comment (anujkumardarji) [reply cypress/e2e/contexts/torrent/specs/list/magnet_link.cy.ts:1]: @josecelano  I reverted changes

**Comment by josecelano** [comment]
ACK 76a626b80fdcd0b256c49ae2dd3c87f8602b4a06

**Review by josecelano — APPROVED** [review]

**Comment by josecelano** [comment]
Thank you, @anujkumardarji, for your first contribution to this repo!!

**Pull request merged**